### ISSUE TITLE
Add a range of TCP functionality for libwallaby

### ISSUE THREAD

**Issue by Zacharyprime** [kipr/libwallaby#131]
Here is a PR for the first attempt at doing this: https://github.com/kipr/libwallaby/pull/73

The KIPR-Update repository is now configured to install a managed mode configurator file. I plan to eventually add the ability to interact with that function to the wifi screen so that the Wombat can connect to wifi sources.

If we do that, TCP (or UDP/other) functionality will be a necessity so that we can add functions such as:

- Allowing IDE use over the internet for virtual learning
- Syncing multiple wombats to one shared network so that there aren't entire APs produced by every Wombat in a room
- Connecting to a Unity Simulator (side project)
- Communicating with another Wombat over TCP (ideally built into libwallaby)
- Allowing the Wombat platform to be used for a broader range of tasks because there is a communication medium between the PC and the Wombat, for example, a browser based remote control robot.

It would also be nice if we could find a way to simplify the TCP connection process, similar to how the create_connect() handles complexity in the background.
```c++
Functions like:
TCP_connect(char* ip, char* port);
TCP_disconnect();
TCP_send(char* data);
```

Here is a link to the Unity Simulator:
https://github.com/Zacharyprime/RoboSim

Here is the TCP code for it:
https://github.com/Zacharyprime/RoboSim/blob/main/RoboSim/Assets/Scripts/Network.cs

(note: the sim is a unfinished, barely even started project as of writing this)

**Comment by Zacharyprime** [comment]
I don't think this is necessary since there are already TCP libraries that exist. This was mostly for the simulator I was creating before we started developing the web based one being used now.

It would be nice to have functions to make it easier to synchronize two wombats by connecting them to each other, but that is too far in the future for this to be on the issues tab.
